ovn-fake-multinode-utils/translate_yaml.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+
+from pathlib import Path
+from dataclasses import dataclass
+from typing import Dict
+import yaml
```

Comment:
This requires installation of `python3-pyyaml` on the orchestrator in the `./do.sh` script.  `install-dependencies.yml` playbook installs it on all other nodes, but it is not executed on orchestrator if orchestrator and the tester are not the same node.